### PR TITLE
Feature/validate criteria mode

### PR DESCRIPTION
**Pull request by bluebill1049**

### CODE REVIEW

File: src/data/api.tsx [hunk 1]
```diff
@@ -315,6 +340,18 @@ export default {
           で詳細を読むことができます。
         </>
       ),
+      validateCriteriaMode: (
```
Comment (bluebill1049) [inline src/data/api.tsx]: @kotarella1110 please help to translate this when you have time. Thank you :)

File: src/data/api.tsx [hunk 2]
```diff
@@ -187,6 +199,18 @@ export default {
           에서 찾아 볼 수 있습니다.
         </>
       ),
+      validateCriteriaMode: (
```
Comment (bluebill1049) [inline src/data/api.tsx]: Hi @adhrinae , please translate this new feature when you have sometimes 🙏

File: src/data/api.tsx (hunk 1 above)
Comment (bluebill1049) [reply src/data/api.tsx]: feel free to improve the English here as well :) @kotarella1110 @adhrinae if it doesn't make sense.

Comment (kotarella1110) [reply src/data/api.tsx]: OK! I will 👍

Comment (bluebill1049) [reply src/data/api.tsx]: thanks @kotarella1110

File: src/data/api.tsx (hunk 2 above)
Comment (adhrinae) [reply src/data/api.tsx]: Sure 👍

**Comment by adhrinae** [comment]
@bluebill1049 Can I merge this PR now?

**Comment by bluebill1049** [comment]
yea sure :) the feature will be released this Friday :)

**Pull request merged**